src/install-update.ts

```diff
@@ -0,0 +1,167 @@
+import { checkForSDKUpdates, Release } from "./check-update.ts";
+import { getJSON } from "./utilities.ts";
+import { projectScripts } from "./mod.ts";
+
+export const SDK_NAME = "the Slack SDK";
```

Comment:
Open to suggestions on if/where to move this (or eliminate). We use this name when relaying information to the CLI about which entity the SDK is, and it's also used for testing purposes for verifying the contents of the response. The `check-update` logic also has this value hard-coded.